unittests.py

```diff
@@ -40,4 +40,8 @@ def run_all_unit_tests():
     return len(test_runner.run(test_suite()).failures) == 0
 
 if __name__ == "__main__":
-    run_all_unit_tests()
+    import sys
+    if run_all_unit_tests():
+        sys.exit(0)
+    else:
+        sys.exit(1)
```

Comment:
ah now I see how this is working...
how about
```
all_tests_ok = run_all_unit_tests()
sys.exit(not all_tests_ok)
```
I would find this a little more explicit

Reply:
Good point, done in a6ad35230277cb82a4a4513fa2c3e37b37db9901.